v2/internal/pkg/imagebuilder/rebuild_catalog.go

```diff
@@ -153,16 +151,12 @@ RUN rm -fr /tmp/cache/* && /bin/opm serve /configs --cache-only --cache-dir=/tmp
 		return catalogCopyRefs, err
 	}
 	manifestPushOptions := manifests.PushOptions{
-		Store:                  buildStore,
-		SystemContext:          destSysContext,
-		ImageListSelection:     cp.CopyAllImages,
-		Instances:              nil,
-		RemoveSignatures:       true,
-		SignBy:                 "",
-		ManifestType:           "application/vnd.oci.image.manifest.v1+json",
-		AddCompression:         []string{},
-		ForceCompressionFormat: false,
-		MaxRetries:             &retries,
+		Store:              buildStore,
+		SystemContext:      destSysContext,
+		ImageListSelection: cp.CopyAllImages,
+		RemoveSignatures:   true,
+		ManifestType:       buildah.OCIv1ImageManifest, //TODO currently red hat catalog is docker v2 format, converting it to oci is not going to cause problems?
```

Comment:
I don't think it will... This is already what v1 does when rebuilding catalogs: upon pulling, it is converted to oci

Reply:
Great, so probably it will not be a problem. Thanks @sherine-k.